pkg/webui/console/components/api-key-form/validation-schema.js

```diff
@@ -15,11 +15,22 @@
 import Yup from '@ttn-lw/lib/yup'
 import sharedMessages from '@ttn-lw/lib/shared-messages'
 
+const pastDateCheck = value => {
+  if (value) {
+    return !(new Date(value) < new Date())
+  }
+
+  return true
+}
+
```

Comment:
```suggestion
const pastDateCheck = value => !value || new Date(value) > new Date()
```
Nit: can be done less verbose